src/rt_core_v2/rttuple.py

```diff
@@ -25,53 +26,55 @@ class TupleType(ValueEnum):
 	D = 'D'
 	F = 'F'
 	NtoDE = 'NtoDE'
-	NtoN = 'NtoN'
-	NtoR = 'NtoR'
+	NtoNTuple = 'NtoN'
+	NtoRTuple = 'NtoRT'
 	NtoC = 'NtoC'
 	NtoLackR = 'NtoR(-)'
 
-#TODO Should polarity be a boolean or an enum? Seems reasonable, but this represents the string output
-class TuplePolarity(ValueEnum):
-	positive = '+'
-	negative = '-'
-
 """Enum representing portions of reality types"""
 class PorType(ValueEnum):
 	singular = '+SU'
 	non_singular = '-SU'
 
+#TODO Move this into the classes, as it is not a semantically sound placement here
 class TupleComponents(enum.Enum):
+	ruit = 'ruit'
+	type = 'type'
 	ar = 'ar'
-	C = 'C'
-	code = 'code'
-	data = 'data'
-	error = 'error'
+	ruip = 'ruip'
+	ruia = 'ruia'
+	unique = 'unique'
+	t = 't'
+	ruid = 'ruid'
 	event = 'event'
 	event_reason = 'event_reason'
-	p_list = 'p_list'
+	td = 'td'
+	replacements = 'replacements'
+	ta = 'ta'
+	C = 'C'
 	polarity = 'polarity'
 	r = 'r'
-	replacements = 'replacements'
-	ruia = 'ruia'
-	ruics = 'ruics'
-	ruid = 'ruid'
-	ruidt = 'ruidt'
+	p_list = 'p_list'
+	rT = 'rT'
+	tr = 'tr'
+	inst = 'inst'
 	ruin = 'ruin'
-	ruins = 'ruins'
-	ruip = 'ruip'
 	ruir = 'ruir'
-	ruit = 'ruit'
-	ruitn = 'ruitn'
-	t = 't'
-	ta = 'ta'
-	td = 'td'
-	tr = 'tr'
-	type = 'type'
-	unique = 'unique'
+	ruics = 'ruics'
+	code = 'code'
+	ruins = 'ruins'
+	data = 'data'
+	ruidt = 'ruidt'
 
 class RtTuple(ABC):
+	"""Abstract Referent Tracking tuple that contains the information that all referent tracking tuples contain
+	
+	Attributes:
+	ruit -- The rui of this tuple
+	type -- The id of the tuple component
+	"""
 	tuple_type = None
-	params = {TupleComponents.ruit:TupleComponents.ruit.value, TupleComponents.type:TupleComponents.type.value}
+	params = {**enum_to_dict({TupleComponents.ruit, TupleComponents.type})}
```

Comment:
This is how I'm enumerating the tuple entries for each class. The ** allows for unpacking a dictionary so that the entries can be added to another. For that instance, it is not necessary, as there is not another dictionary that is being merged with it. It is done purely to be consistent in format with the other tuples, but I'd be happy to change it.

I felt that calling the helper function looks cleaner than having each entry being in the form:
 "TupleComponents.x : TupleComponents.x.value